File: codeforces/1A.rb
# Codeforces 1A
# TAG: Math
# NOTE: In programming language with `floored division`(Python, Ruby), `a/b = floor(a/b)`.
#       Therefore `ceiling(a/b) = -(-a/b)`.
# REF: https://en.wikipedia.org/wiki/Modulo_operation
# STATUS: AC/2226629/1348229339/Ruby/108ms/5500KB

n, m, a = gets.split.map(&:to_i)
puts (-n/a)*(-m/a)

File: codeforces/1A.cpp
/*
 * Codeforces 1A
 * TAG: Math
 * NOTE: `ceiling(a/b) = (a+b-1)/b` if `a > 0 and b > 0`
 *       Choose proper variable type by estimating range of values.
 * STATUS: AC/2226754/1348231145/C++/31ms/1500KB
*/

#include <iostream>

int main()
{
    unsigned long long n, m, a;

    std::cin >> n >> m >> a;
    std::cout << ((n+a-1)/a)*((m+a-1)/a) << std::endl;

    return 0;
}
